src/main/java/org/prebid/server/handler/openrtb2/AmpHandler.java

```diff
@@ -165,7 +166,9 @@ private AmpResponse toAmpResponse(BidRequest bidRequest, BidResponse bidResponse
         final ExtResponseDebug extResponseDebug = Objects.equals(bidRequest.getTest(), 1)
                 ? extResponseDebugFrom(bidResponse) : null;
 
-        return AmpResponse.of(targeting, extResponseDebug);
+        final Map<String, List<BidderErrorInfo>> errors = errorsFrom(bidResponse);
```

Comment:
`errors` field in response should be filled only if request asked for debug information in response